src/__init__.py

```diff
@@ -0,0 +1,25 @@
+from flask import Flask
+from flask_pymongo import PyMongo
+from flask_admin import Admin
+from flask_login import LoginManager
+
+# flask
+app = Flask(__name__)
+app.config.from_pyfile("config.py")
+
+# mongo db
+mongo = PyMongo(app)
+
+# login manager
+login_manager = LoginManager()
+login_manager.init_app(app)
+login_manager.login_view = 'login'
+login_manager.login_message = u"Please login to access this page."
+login_manager.login_message_category = "warning"
+
+# Create admin
+admin = Admin(app, name='VUSualizer admin', template_mode='bootstrap3')
+
+
+from . import *
+from .views import main, auth, admin_view
```

Comment:
flake8 detects a few issues regarding the import statement.

./src/__init__.py:24:1: E402 module level import not at top of file
./src/__init__.py:24:1: F403 'from . import *' used; unable to detect undefined names
./src/__init__.py:24:1: F401 'from . import *' imported but unused
./src/__init__.py:25:1: E402 module level import not at top of file
./src/__init__.py:25:1: F401 '.views.main' imported but unused
./src/__init__.py:25:1: F401 '.views.auth' imported but unused
./src/__init__.py:25:1: F401 '.views.admin_view' imported but unused

Reply:
removed the import * 👍 
however kept de .views at the end of the file instead of beginning. Need to be loaded after creating the flask app. these are also not "unused" but refer to the files in de "views" folder (main.py, auth.py and admin_view.py).

I added a comment to prevent this from happening again according to the flake8 documentation
[flake documentation](https://flake8.pycqa.org/en/stable/user/violations.html#in-line-ignoring-errors)